### PR TITLE
bpo-46541: remove usage of _Py_IDENTIFIER from _ssl module

### PR DESCRIPTION
**Pull request by kumaraditya303**
https://bugs.python.org/issue46541

### CODE REVIEW

**Review by erlend-aasland — APPROVED** [review]
lgtm

**Comment by corona10** [comment]
I assigned core devs whom highly involved in this issue.

**Comment by dwvshell** [comment]
Good Afternnon

File: Modules/_ssl.c
Comment (tiran) [inline Modules/_ssl.c]: These attributes are simple types that cannot reference other objects. There is no need to visit them.

**Review by tiran — CHANGES_REQUESTED** [review]

**Comment by kumaraditya303** [comment]
I have made the requested changes; please review again

File: Modules/_ssl.c
Comment (kumaraditya303) [reply Modules/_ssl.c]: Fixed

**Review by corona10 — APPROVED** [review]
LGTM
see my decision: https://github.com/python/cpython/pull/31475#issuecomment-1055508822
I am waiting for @tiran 's decision since he is the maintainer of the SSL module.

**Review by tiran — APPROVED** [review]

**Pull request merged**
